tests/unit/plugins/modules/test_vault_kv2_delete.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2022 Brian Scholer (@briantist)
```

Comment:
I know it's a WIP but update this with your info.

And the method names will need to change from the kv2_get names as well.

Reply:
done :)

Reply:
Method names look good, but it's still my name in the copyright ;)